sdks/java/io/redis/src/main/java/org/apache/beam/sdk/io/redis/RedisIO.java

```diff
@@ -199,6 +209,10 @@ public Read withBatchSize(int batchSize) {
       return builder().setBatchSize(batchSize).build();
     }
 
+    public Read enableSSL() {
```

Comment:
Remove, users wanting to do this can do it via the `ConnectionConfiguration` object.